pkg/base/write.go

```diff
@@ -132,7 +132,6 @@ func (b *Base) writeBase(options WriteOptions) error {
 			APIVersion: "kustomize.config.k8s.io/v1beta1",
 			Kind:       "Kustomization",
 		},
-		Namespace:             b.Namespace,
```

Comment:
we should keep this since it can affect non-helm resources. you probably want to get rid of this instead: https://github.com/replicatedhq/kots/blob/b1dd65d608efab4c02d51c97ca26e407e8e5aad4/pkg/base/replicated.go#L124